components/advanced/babel-plugin.js

```diff
@@ -163,7 +164,7 @@ const BabelPlugin = () =>
         literals down to a smaller representation than what Babel normally does,
         because <Code>styled-components</Code> template literals don't need to
         be 100% spec compliant. see{' '}
-        <a href="#tagged-template-literals">Tagged Template Literals</a> for
+        <Link href="#tagged-template-literals">Tagged Template Literals</Link> for
```

Comment:
Sorry for not being clear enough on this :( You'll have to pass the `inline` prop for it to show up with the correct styling: https://github.com/styled-components/styled-components-website/blob/master/components/api/helpers/with-theme.js#L52